### PR TITLE
fix: export "Deno.memoryUsage()" in worker context

### LINKED ISSUE #13245: `Deno.memoryUsage()` is not a function in Worker context
```
error: Uncaught (in worker "WebSocket Server") (in promise) TypeError: Deno.memoryUsage is not a function
```
It seems like workers don't have this api.
Other api's and the deno namespace are available though
I'm not sure if this is intentional or not but I would really like to have this function exposed to workers aswell.

### PR DESCRIPTION
**Pull request by R0dn3yS**
closes #13245

### CODE REVIEW

**Review by bartlomieju — CHANGES_REQUESTED** [review]
Thank you for PR @R0dn3yS, can you please add a test to `cli/tests/testdata/workers/test.ts` that verifies that this API is present?

**Comment by R0dn3yS** [comment]
@bartlomieju PTAL

File: cli/tests/testdata/workers/test.ts [hunk 1]
```diff
@@ -798,3 +798,23 @@ Deno.test({
     worker.terminate();
   },
 });
+
+Deno.test({
+  name: "worker Deno.memoryUsage",
+  fn: async function () {
+    const w = new Worker(
+      "data:application/typescript;base64,c2VsZi5vbm1lc3NhZ2UgPSBmdW5jdGlvbigpIHtzZWxmLnBvc3RNZXNzYWdlKERlbm8ubWVtb3J5VXNhZ2UoKSl9",
```
Comment (bartlomieju) [inline cli/tests/testdata/workers/test.ts:806]: Please add a comment above that shows original source

File: cli/tests/testdata/workers/test.ts [hunk 2]
```diff
@@ -798,3 +798,23 @@ Deno.test({
...
+    );
+
+    w.postMessage(null);
+
+    const memoryUsagePromise = deferred();
+    w.onmessage = function (evt) {
+      memoryUsagePromise.resolve(evt.data);
+    };
+
+    console.log(await memoryUsagePromise);
```
Comment (bartlomieju) [inline cli/tests/testdata/workers/test.ts:817]: This is error prone, please use assertion function here, that check that received message has all the fields from `Deno.memoryUsage()`

**Review by bartlomieju — COMMENTED** [review]
@R0dn3yS nice work, this is almost ready

**Comment by R0dn3yS** [comment]
@bartlomieju should be good now

**Review by bartlomieju — APPROVED** [review]
LGTM, thank you @R0dn3yS

**Pull request merged**